engines/sci/graphics/menu.cpp

```diff
@@ -787,10 +854,22 @@ GuiMenuItemEntry *GfxMenu::interactiveWithKeyboard() {
 					}
 					break;
 				case kSciKeyLeft:
-					newMenuId--; newItemId = 1;
+					if (!g_sci->isLanguageRTL()) {
+						newMenuId--;
+						newItemId = 1;
+					} else {
+						newMenuId++;
+						newItemId = 1;
+					}
 					break;
 				case kSciKeyRight:
-					newMenuId++; newItemId = 1;
+					if (!g_sci->isLanguageRTL()) {
+						newMenuId++;
+						newItemId = 1;
+					} else {
+						newMenuId--;
+						newItemId = 1;
+					}
```

Comment:
There's no need for a new block here. You can multiply with a value, e.g:

    int multiplier = !g_sci->isLanguageRTL() ? 1 : -1;
    ....
    newMenuId -= 1 * multiplier;
    newItemId = 1;
    ....
    newMenuId += 1 * multiplier;
    newItemId = 1;